lib/emittr/events.rb

```diff
@@ -39,18 +39,37 @@ def once(event, &block)
         on(event, &off_block)
       end
 
+      def on_any(&block)
```

Comment:
for me, `any` is more descriptive, but since this will be used internally only, `*` can do the job